### PR TITLE
new icon: vscode-insiders (original)

### PR DESCRIPTION
**Pull request by theblobscp**
I hope I did this right

### CODE REVIEW

File: devicon.json [hunk 1]
```diff
@@ -4287,6 +4287,18 @@
...
     },
+    {
+        "name": "vscode-insiders",
+        "tags":[
+            "editor", "ide"
+        ],
+        "verisons": {
+            "svg": ["original"],
+            "font": "plain"
+        },
```
Comment (Panquesito7) [inline devicon.json:4298]:
```suggestion
        "versions": {
            "svg": [
                "original",
                "plain"
            ],
            "font": [
                "original"
            ]
        },
```

File: devicon.json [hunk 2]
```diff
@@ -4287,6 +4287,18 @@
...
+        "color": "#03b293",
+        "aliases": []
```
Comment (Panquesito7) [inline devicon.json:4300]:
```suggestion
        "aliases": [
            {
                "base": "original",
                "alias": "plain"
            }
        ]
```

File: icons/vscode-insiders/vscode-insiders-original.svg [hunk 3]
```diff
@@ -0,0 +1 @@
...309.375m5.968.238l-.367.399.399-.367c.37-.34.449-.43.367-.43-.016 0-.196.18-.399.398m1.578 4.688c.133.027.348.027.48 0 .134-.024.024-.047-.241-.047-.262 0-.371.023-.239.047" fill="#039b7a"/><path d="M92.535.504c-.969.133-2.559.715-3.094 1.133-.168.129-.066.12.48-.051 2.403-.75 5.094.617 5.895 3 .145.426.164 7.125.195 59.359l.032 58.883-.242.684c-.891 2.523-3.825 3.906-6.153 2.902-.8-.344-.265.043.664.476 2.84 1.336 4.414 1.11 9.637-1.402 1.79-.863 3.684-1.77 4.21-2.02.528-.253 1.634-.78 2.454-1.175 3.379-1.63 6.617-3.184 9.652-4.637 9.438-4.52 9.79-4.758 11-7.43.746-1.64.68 2.746.68-46.12 0-49.704.094-44.704-.855-46.595-1.145-2.28-1.957-2.863-7.942-5.703a283.35 283.35 0 01-2.613-1.25c-.883-.422-2.13-1.023-2.774-1.332-.648-.308-1.941-.933-2.882-1.386a1108.332 1108.332 0 00-5.493-2.637C95.683.55 96.16.765 94.886.539c-.613-.11-1.703-.125-2.351-.035" fill="#1bc2a3"/><path d="M96.051 105.44c0 6.219.012 8.777.027 5.688s.016-8.18 0-11.31c-.015-3.128-.027-.597-.027 5.622" fill="#1bb39c"/><path d="M1.54 38.961l-.313.344.347-.313c.192-.172.348-.328.348-.343 0-.086-.09-.012-.383.312m84.836 84.207c.129.027.348.027.477 0 .132-.023.027-.047-.239-.047s-.37.024-.238.047" fill="#03a384"/></g></svg>
```
Comment (Panquesito7) [inline icons/vscode-insiders/vscode-insiders-original.svg:1]:
```suggestion
<svg width="128" height="128" version="1.0" viewBox="0 0 128 128" xmlns="http://www.w3.org/2000/svg"><g fill-rule="evenodd"><path d="M11.625 31.047a4.807 4.807 0 00-2.047.77c-.578.37-7.465 6.59-8.258 7.453-1.676 1.836-1.683 5.082-.015 6.93.996 1.105 83.152 76.292 83.648 76.554 1.613.852 4.172.098 5.11-1.507.636-1.086.605-.25.605-16.23v-14.63l-.293-.269c-.164-.152-2.961-2.34-6.215-4.863-3.258-2.524-10.68-8.278-16.496-12.79a38776.638 38776.638 0 01-33.637-26.09c-10.465-8.14-18.652-14.46-18.988-14.663-.903-.547-2.324-.825-3.414-.664" fill="#03b293"/><path d="M93.469.395c.16.027.402.027.531 0 .133-.024 0-.043-.293-.043s-.402.02-.238.043m1.09 2.378c.23.235.441.426.472.426.028 0-.136-.191-.363-.426-.23-.234-.441-.425-.472-.425-.028 0 .136.191.363.425m-8.504 2.141c-1.09.176-1.692.578-3.656 2.438-.41.39-1.32 1.23-2.024 1.867-.707.64-1.617 1.48-2.027 1.867-.41.387-1.324 1.23-2.027 1.867a175.94 175.94 0 00-2.028 1.863c-.41.391-1.297 1.207-1.972 1.82a149.41 149.41 0 00-1.906 1.762c-.372.352-.731.692-.801.75a236.982 236.982 0 00-4.051 3.73 282.07 282.07 0 00-2.898 2.657c-.06.067-.372.356-.692.645-.324.289-.851.77-1.172 1.066-.324.3-1.5 1.383-2.613 2.402a281.015 281.015 0 00-3.398 3.153c-.461.402-2.817 2.558-3.43 3.14-.41.387-1.32 1.227-2.027 1.864-.704.64-1.614 1.48-2.028 1.867-.41.39-1.32 1.23-2.023 1.87-.707.641-1.809 1.657-2.453 2.25-.649.599-2.133 1.966-3.309 3.04-1.172 1.078-2.144 1.984-2.16 2.02-.016.035 4.184 3.331 9.332 7.328 11.543 8.953 11 8.535 10.78 8.3-.198-.214-.702.22 5.34-4.57 1.196-.95 2.352-1.867 2.567-2.043.219-.176.559-.45.762-.61 1.476-1.163 3.254-2.566 3.59-2.835.558-.453 3.195-2.543 3.886-3.082.649-.508 5.946-4.707 6.594-5.23a892.337 892.337 0 005.656-4.477c.793-.633 1.13-.899 1.668-1.32.344-.27 1.211-.958 1.93-1.528l1.309-1.043V23.265c0-10.004-.035-14.676-.118-15.109a4.013 4.013 0 00-4.601-3.242m39.547 10.445c.226.234.437.426.469.426.027 0-.133-.192-.364-.426s-.441-.426-.468-.426c-.032 0 .132.192.363.426M8.152 32.879l-.313.348.348-.313c.191-.172.348-.328.348-.348 0-.082-.09-.007-.383.313M6.98 33.945l-.313.348.344-.313c.192-.171.348-.328.348-.347 0-.082-.086-.008-.379.312m-1.172 1.067l-.316.347.347-.312c.192-.172.348-.328.348-.348 0-.082-.09-.008-.379.313m-1.227 1.12l-.367.403.399-.371c.218-.2.398-.379.398-.398 0-.082-.09-.004-.43.367m-1.175 1.066l-.368.403.403-.372c.37-.34.45-.43.367-.43-.02 0-.2.18-.402.4m-1.172 1.066l-.367.402.398-.367c.375-.344.45-.434.367-.434-.015 0-.195.18-.398.399m.215 9.12c.261.278.488.509.507.509.079 0-.007-.098-.476-.54l-.508-.476.477.508m1.336 1.254c.23.234.44.425.472.425.028 0-.137-.191-.363-.425-.23-.235-.442-.426-.473-.426-.027 0 .137.191.364.426m1.222 1.093c.442.47.54.555.54.473 0-.016-.227-.242-.509-.504l-.503-.476.472.507m1.281 1.172c.262.278.489.508.508.508.078 0-.004-.098-.476-.539l-.508-.477.476.508m1.336 1.254c.23.235.442.426.473.426.027 0-.137-.191-.363-.426-.23-.234-.442-.426-.473-.426-.027 0 .137.192.363.426m1.336 1.227c.2.207.387.375.414.375.032 0-.11-.168-.308-.375-.2-.207-.387-.375-.414-.375-.032 0 .109.168.308.375m1.278 1.172c.199.207.386.375.417.375.028 0-.109-.168-.308-.375-.2-.203-.387-.371-.418-.371-.027 0 .11.168.309.37m1.28 1.176c.2.204.387.372.415.372.031 0-.11-.168-.309-.372-.199-.207-.383-.375-.414-.375-.027 0 .11.168.309.375m1.28 1.172c.2.207.388.375.415.375.031 0-.11-.168-.309-.375-.199-.203-.386-.37-.414-.37-.03 0 .11.167.309.37m1.227 1.121c.226.235.44.426.468.426.032 0-.133-.191-.363-.426-.227-.234-.441-.425-.469-.425-.031 0 .133.191.364.425m1.332 1.227c.199.203.386.371.414.371.03 0-.11-.168-.309-.37-.2-.208-.383-.376-.414-.376-.027 0 .11.168.309.375m1.066.938c0 .02.156.175.348.347l.347.313-.312-.344c-.293-.324-.383-.398-.383-.316m1.281 1.175c0 .02.156.176.348.348l.344.313-.313-.348c-.293-.324-.379-.395-.379-.313m1.492 1.407c.2.207.387.375.414.375.032 0-.109-.168-.308-.375-.196-.207-.383-.375-.414-.375-.028 0 .11.168.308.375m1.282 1.171c.199.208.386.375.414.375.03 0-.11-.167-.309-.374-.2-.204-.387-.372-.414-.372-.031 0 .11.168.309.371m1.28 1.176c.2.203.384.371.415.371.027 0-.11-.168-.309-.37-.199-.208-.387-.376-.414-.376-.031 0 .11.168.309.375m1.277 1.172c.2.207.387.375.414.375.031 0-.11-.168-.305-.375-.199-.203-.386-.37-.418-.37-.027 0 .11.167.309.37m-6.09 1.942c-.726.625-1.71 1.48-2.183 1.894-.477.414-1.649 1.422-2.602 2.246-.957.82-2.324 1.996-3.035 2.613-.715.614-2.11 1.813-3.098 2.665-3.808 3.28-4.601 3.98-4.863 4.304-1.442 1.778-1.578 4.403-.332 6.317.344.535 6.226 6.035 8.304 7.77.946.792 2.665 1.234 4.008 1.03 1.184-.18 1.957-.636 5.625-3.32 1.848-1.355 4.028-2.95 4.848-3.547l2.984-2.187c.829-.602 1.618-1.18 1.754-1.278.14-.101.38-.28.535-.402.153-.121.711-.527 1.239-.902a26.195 26.195 0 001.078-.793c.066-.063.894-.664 1.84-1.34.945-.672 1.707-1.258 1.695-1.297-.043-.113-16.312-14.914-16.395-14.914-.043 0-.672.512-1.402 1.14m7.425-.714c.227.234.438.426.47.426.027 0-.134-.192-.364-.426s-.441-.426-.469-.426c-.031 0 .133.191.363.426m1.332 1.226c.2.207.387.375.415.375.03 0-.11-.168-.31-.375-.198-.203-.386-.37-.413-.37-.031 0 .11.167.308.37m1.227 1.122c.23.234.441.425.469.425.031 0-.133-.191-.363-.425-.227-.235-.442-.426-.47-.426-.03 0 .134.191.364.426m1.281 1.171c.227.235.438.426.469.426.027 0-.133-.191-.363-.426-.23-.234-.442-.425-.469-.425-.031 0 .133.191.363.425m1.278 1.176c.23.235.441.426.472.426.028 0-.137-.191-.363-.426-.23-.234-.441-.43-.473-.43-.027 0 .137.196.364.43m1.28 1.172c.231.234.442.426.47.426.03 0-.133-.192-.364-.426-.226-.234-.441-.426-.469-.426-.03 0 .133.192.364.426m1.281 1.172c.227.234.437.43.469.43.027 0-.133-.196-.364-.43-.23-.234-.441-.426-.468-.426-.032 0 .132.192.363.426m1.332 1.227c.2.207.387.375.414.375.031 0-.11-.168-.309-.375-.199-.204-.382-.371-.414-.371-.027 0 .11.167.309.37m1.281 1.176c.2.204.383.371.414.371.028 0-.11-.167-.308-.37-.2-.208-.387-.376-.414-.376-.032 0 .109.168.308.375m1.277 1.172c.2.207.387.375.418.375.028 0-.109-.168-.308-.375-.2-.203-.387-.37-.418-.37-.027 0 .11.167.308.37m1.282 1.172c.199.207.386.375.414.375.031 0-.11-.168-.309-.375-.2-.203-.383-.37-.414-.37-.027 0 .11.167.309.37m1.226 1.121c.23.235.442.426.469.426.031 0-.133-.191-.363-.426-.227-.234-.438-.425-.469-.425-.027 0 .133.19.363.425m1.332 1.227c.2.207.387.375.418.375.028 0-.11-.168-.308-.375-.2-.203-.387-.371-.418-.371-.028 0 .109.168.308.37m1.067.942c0 .02.156.176.347.348l.348.312-.313-.347c-.293-.324-.382-.395-.382-.313m1.441 1.352c.23.234.441.43.469.43.031 0-.133-.196-.363-.43-.227-.234-.438-.426-.47-.426-.027 0 .134.192.364.426m1.332 1.227c.2.207.387.375.418.375.027 0-.11-.168-.309-.375-.199-.204-.386-.372-.418-.372-.027 0 .11.168.31.372m1.28 1.175c.2.203.387.371.414.371.032 0-.11-.168-.308-.37-.2-.208-.383-.376-.414-.376-.028 0 .109.168.308.375m1.281 1.172c.2.207.387.375.415.375.03 0-.11-.168-.31-.375-.198-.203-.386-.37-.413-.37-.031 0 .11.167.308.37m1.278 1.176c.199.203.386.371.418.371.027 0-.11-.168-.309-.371-.2-.207-.387-.375-.418-.375-.027 0 .11.168.309.375m1.336 1.227c.226.234.441.425.468.425.032 0-.132-.191-.363-.425-.23-.239-.441-.43-.469-.43-.03 0 .133.191.364.43m41.918 14.984c0 8.097.011 11.426.027 7.394.016-4.027.016-10.652 0-14.719-.016-4.066-.027-.77-.027 7.324M50.13 91.148c.199.203.386.371.414.371.031 0-.11-.168-.309-.37-.2-.208-.383-.376-.414-.376-.027 0 .11.168.309.375m1.226 1.118c.23.234.442.43.473.43.027 0-.137-.196-.367-.43-.227-.235-.438-.426-.469-.426-.027 0 .137.191.363.426m1.281 1.175c.227.235.442.426.47.426.03 0-.134-.191-.364-.426-.23-.234-.441-.43-.469-.43-.031 0 .133.196.363.43m1.282 1.172c.226.235.437.426.468.426.028 0-.136-.191-.363-.426-.23-.234-.441-.426-.469-.426-.03 0 .133.192.364.426m1.277 1.172c.23.234.441.43.473.43.027 0-.137-.196-.368-.43-.226-.234-.437-.426-.468-.426-.028 0 .136.192.363.426m1.336 1.227c.2.207.383.375.414.375.027 0-.11-.168-.309-.375-.199-.203-.386-.371-.414-.371-.031 0 .11.168.309.37m1.277 1.177c.2.203.387.37.414.37.031 0-.11-.167-.304-.37-.2-.208-.387-.375-.418-.375-.028 0 .109.167.308.375m1.281 1.171c.2.207.387.375.414.375.032 0-.109-.168-.308-.375-.2-.203-.387-.37-.414-.37-.031 0 .11.167.308.37m1.282 1.176c.199.203.382.371.414.371.027 0-.11-.168-.309-.37-.2-.208-.387-.376-.414-.376-.031 0 .11.168.309.375m1.226 1.117c.227.235.438.426.469.426.027 0-.133-.191-.363-.426s-.442-.425-.47-.425c-.03 0 .134.191.364.425m1.277 1.176c.23.235.442.426.473.426.027 0-.137-.191-.363-.426-.23-.234-.442-.43-.473-.43-.027 0 .137.196.363.43m1.336 1.227c.2.203.383.37.414.37.028 0-.109-.167-.308-.37-.2-.207-.387-.375-.414-.375-.031 0 .11.168.308.375m1.067.937c0 .02.156.176.343.348l.348.312-.312-.347c-.293-.32-.38-.395-.38-.313m1.493 1.407c.2.207.387.375.414.375.031 0-.11-.168-.309-.375-.199-.204-.386-.372-.414-.372-.03 0 .11.168.309.372m1.281 1.175c.2.203.383.371.414.371.028 0-.11-.168-.308-.37-.2-.208-.387-.376-.414-.376-.032 0 .109.168.308.375m1.277 1.172c.2.207.387.375.418.375.028 0-.109-.168-.308-.375-.2-.203-.387-.37-.418-.37-.027 0 .11.167.308.37m1.282 1.176c.199.203.386.371.414.371.031 0-.11-.168-.309-.371-.2-.207-.383-.375-.414-.375-.027 0 .11.168.309.375m1.28 1.172c.2.203.384.371.415.371s-.11-.168-.309-.371c-.199-.207-.386-.375-.414-.375-.031 0 .11.168.309.375m1.387 1.281c.199.203.382.371.414.371.027 0-.11-.168-.309-.37-.2-.208-.387-.376-.414-.376-.031 0 .11.168.309.375m1.277 1.172c.2.207.387.375.418.375.027 0-.11-.168-.309-.375s-.386-.375-.418-.375c-.027 0 .11.168.309.375m1.226 1.121c.23.234.442.426.473.426.027 0-.137-.192-.363-.426-.23-.234-.442-.43-.473-.43-.027 0 .137.196.363.43m1.282 1.172c.23.234.441.426.468.426.032 0-.132-.192-.363-.426-.226-.234-.437-.426-.469-.426-.027 0 .133.192.364.426m1.28 1.172c.227.234.442.43.47.43.03 0-.133-.196-.364-.43s-.441-.426-.468-.426c-.032 0 .132.192.363.426m1.277 1.176c.23.234.442.425.473.425.027 0-.137-.19-.363-.425-.23-.235-.442-.426-.473-.426-.027 0 .137.191.363.426m1.336 1.226c.2.203.387.371.414.371.031 0-.11-.168-.308-.37-.2-.208-.387-.376-.415-.376-.03 0 .11.168.309.375m1.277 1.172c.2.207.387.375.418.375.028 0-.11-.168-.308-.375-.2-.203-.387-.37-.418-.37-.028 0 .11.167.308.37m1.282 1.176c.199.203.386.371.414.371.03 0-.11-.168-.309-.371-.2-.207-.383-.375-.414-.375-.027 0 .11.168.309.375m5.968.238l-.367.399.399-.367c.37-.34.449-.43.367-.43-.016 0-.196.18-.399.398m1.578 4.688c.133.027.348.027.48 0 .134-.024.024-.047-.241-.047-.262 0-.371.023-.239.047" fill="#039b7a"/><path d="M92.535.504c-.969.133-2.559.715-3.094 1.133-.168.129-.066.12.48-.051 2.403-.75 5.094.617 5.895 3 .145.426.164 7.125.195 59.359l.032 58.883-.242.684c-.891 2.523-3.825 3.906-6.153 2.902-.8-.344-.265.043.664.476 2.84 1.336 4.414 1.11 9.637-1.402 1.79-.863 3.684-1.77 4.21-2.02.528-.253 1.634-.78 2.454-1.175 3.379-1.63 6.617-3.184 9.652-4.637 9.438-4.52 9.79-4.758 11-7.43.746-1.64.68 2.746.68-46.12 0-49.704.094-44.704-.855-46.595-1.145-2.28-1.957-2.863-7.942-5.703a283.35 283.35 0 01-2.613-1.25c-.883-.422-2.13-1.023-2.774-1.332-.648-.308-1.941-.933-2.882-1.386a1108.332 1108.332 0 00-5.493-2.637C95.683.55 96.16.765 94.886.539c-.613-.11-1.703-.125-2.351-.035" fill="#1bc2a3"/><path d="M96.051 105.44c0 6.219.012 8.777.027 5.688s.016-8.18 0-11.31c-.015-3.128-.027-.597-.027 5.622" fill="#1bb39c"/><path d="M1.54 38.961l-.313.344.347-.313c.192-.172.348-.328.348-.343 0-.086-.09-.012-.383.312m84.836 84.207c.129.027.348.027.477 0 .132-.023.027-.047-.239-.047s-.37.024-.238.047" fill="#03a384"/></g></svg>
```

**Review by Panquesito7 — CHANGES_REQUESTED** [review]

**Review by Panquesito7 — CHANGES_REQUESTED** [review]
That part of the icon looks kind of separated from the front part. Please fix that.

image

**Comment by theblobscp** [comment]
> That part of the icon looks kind of separated from the front part. Please fix that.
> 
> image

It's supposed to be that way. Example:
image
As you can tell, there is a slight spacing in the icon

**Comment by Panquesito7** [comment]
Now that I searched a bit more about the icon, I think it's best to close this PR (see the Visual Studio Code [**branding guidelines**](https://code.visualstudio.com/brand)).

**Comment by theblobscp** [comment]
> Now that I searched a bit more about the icon, I think it's best to close this PR (see the Visual Studio Code [**branding guidelines**](https://code.visualstudio.com/brand)).

Oh yeah. Alright, have a nice day

**Pull request closed without merging**